BraveRewardsUI/QA Settings/QASettingsViewController.swift

```diff
@@ -239,6 +239,9 @@ public class QASettingsViewController: TableViewController {
     guard let appSupportPath = NSSearchPathForDirectoriesInDomains(.applicationSupportDirectory, .userDomainMask, true).first else { return }
     let dbPath = (appSupportPath as NSString).appendingPathComponent("ledger/Rewards.db")
     let activity = UIActivityViewController(activityItems: [URL(fileURLWithPath: dbPath)], applicationActivities: nil)
+    if UIDevice.current.userInterfaceIdiom == .pad {
+      activity.popoverPresentationController?.sourceView = view
```

Comment:
would love to see an xcode compile warning when this is not provided, or make it required somehow

Reply:
Yeah crazy how many times we get dinged by this…